app/src/test/java/io/intrepid/contest/screens/contestcreation/NewContestPresenterTest.java

```diff
@@ -22,6 +22,8 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+/*Tests the navigation flow for entering a new contest and tests that the Activity, though this presenter
```

Comment:
This comment is probably not needed, I think its clear from the tests what is happening.